.circleci/config.yml

```diff
@@ -0,0 +1,17 @@
+version: 2.1
+orbs:
+  node: circleci/node@1.1.6
+jobs:
+  build-and-test:
+    executor:
+      name: node/default
+    steps:
+      - checkout
+      - node/with-cache:
+          steps:
+            - run: npm install
+            - run: npm test
```

Comment:
Shouldn't matter too much, but this project is using yarn. Also, `npm ci` is faster.